payment_method.go

```diff
@@ -0,0 +1,115 @@
+}
+
+type ActionType string
+
+const (
+	Auth       ActionType = "AUTH"
+	ResendAuth ActionType = "RESEND_AUTH"
+	Capture    ActionType = "CAPTURE"
+	Cancel     ActionType = "CANCEL"
+)
+
+type URLType string
+
+const (
+	API      URLType = "API"
+	Web      URLType = "WEB"
+	Mobile   URLType = "MOBILE"
+	Deeplink URLType = "DEEPLINK"
+)
+
+type Metadata map[string]interface{}
+
+type BillingInformation struct {
+	Country       constant.CountryEnum `json:"country"`
+	StreetLine1   *string              `json:"street_line1"`
+	StreetLine2   *string              `json:"street_line2"`
+	City          *string              `json:"city"`
+	ProvinceState *string              `json:"province_state"`
+	PostalCode    *string              `json:"postal_code"`
+}
+
+type EwalletMethod struct {
+	ChannelCode       ewallet.ChannelCode       `json:"channel_code"`
+	ChannelProperties ewallet.ChannelProperties `json:"channel_properties"`
+	Account           ewallet.Account           `json:"account"`
+}
+
+type DirectDebitMethod struct {
+	LinkedAccountTokenID string                        `json:"-"`
+	LinkedAccountID      string                        `json:"-"`
+	ChannelCode          directdebit.ChannelCode       `json:"channel_code"`
+	ChannelProperties    directdebit.ChannelProperties `json:"channel_properties"`
+	Type                 directdebit.Type              `json:"type"`
+	BankAccount          *directdebit.BankAccount      `json:"bank_account"`
+	DebitCard            *directdebit.DebitCard        `json:"debit_card"`
+}
+
+type OverTheCounterMethod struct {
+	Amount            float64                          `json:"amount"`
+	Currency          constant.CurrencyEnum            `json:"currency"`
+	ChannelCode       overthecounter.ChannelCode       `json:"channel_code"`
+	ChannelProperties overthecounter.ChannelProperties `json:"channel_properties"`
+}
+
+type VirtualAccountMethod struct {
+	Amount            float64                          `json:"amount"`
+	MinimumAmount     float64                          `json:"min_amount,omitempty"`
+	MaximumAmount     float64                          `json:"max_amount,omitempty"`
+	Currency          constant.CurrencyEnum            `json:"currency"`
+	ChannelCode       virtualaccount.ChannelCode       `json:"channel_code"`
+	ChannelProperties virtualaccount.ChannelProperties `json:"channel_properties"`
+}
+
+type QRCodeMethod struct {
+	Amount            float64                  `json:"amount"`
+	Currency          constant.CurrencyEnum    `json:"currency"`
+	ChannelCode       qrcode.ChannelCode       `json:"channel_code"`
+	ChannelProperties qrcode.ChannelProperties `json:"channel_properties"`
+}
+
+type PaymentMethodResponse struct {
+	ID                 string                           `json:"id"`
+	Type               constant.PaymentMethodTypeEnum   `json:"type"`
+	Country            constant.CountryEnum             `json:"country"`
+	MerchantID         string                           `json:"business_id"`
```

Comment:
let's call this `BusinessID` so it matches the json and it's also called `BusinessID` elsewhere in the repo

eg https://github.com/xendit/xendit-go/blob/c12673fe6f1a9b9233598466f16c48f927e4bc28/card.go#L15